src/fusor/harvester.py

```diff
@@ -368,5 +372,6 @@ def load_records(self) -> list[CategoricalFusion]:
             if moa_fusion:
                 translated_fusions.append(moa_fusion)
         self._count_dropped_fusions(moa_fusions, translated_fusions)
+        self.translated_fusions = translated_fusions
```

Comment:
I take back this comment. I think it's fine not to store as instance var for now.